### PR TITLE
gnuykeaj theme

### PR DESCRIPTION
**Pull request by andyklimczak**
:see_no_evil: :hankey: 

Should be much better now !

### CODE REVIEW

File: themes/gnuykeaj/fish_prompt.fish [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  set -l cyan (set_color cyan)
+  set -l yellow (set_color yellow)
+  set -l red (set_color red)
+  set -l blue (set_color blue)
+  set -l green (set_color green)
+  set -l normal (set_color normal)
+
+  set -l cwd $blue(basename (pwd | sed "s:^$HOME:~:"))
+
+  # Output the prompt, left to right
```
Comment (bpinto) [inline themes/gnuykeaj/fish_prompt.fish]: What is this for?

Comment (andyklimczak) [inline themes/gnuykeaj/fish_prompt.fish]: Just a comment from the theme that I used to make this. I suppose he was very specific, and didn't want to confuse anyone. Want me to take it out? :penguin:

Comment (bpinto) [inline themes/gnuykeaj/fish_prompt.fish]: Seems like this comment is lost in the code, I think it's better to remove
it.

**Comment by andyklimczak** [comment]
Removed the comment !
:cocktail: :crown:

**Comment by bpinto** [comment]
Yay!

**Pull request merged**